docs/reference/cluster_manifest.md

```diff
@@ -173,6 +173,20 @@ These parameters are grouped directly under  the `spec` key in the manifest.
   [administrator docs](https://github.com/zalando/postgres-operator/blob/master/docs/administrator.md#load-balancers-and-allowed-ip-ranges)
   for more information regarding default values and overwrite rules.
 
+* **masterServiceAnnotations**
+  A map of key value pairs that gets attached as [annotations](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/)
+  to the master services created for the database cluster. Check the
+  [administrator docs](https://github.com/zalando/postgres-operator/blob/master/docs/administrator.md#load-balancers-and-allowed-ip-ranges)
+  for more information regarding default values and overwrite rules.
+  This field takes precedence over `serviceAnnotations` for the master services if not empty.
```

Comment:
I guess singular is better: "to the master service of ..." (same for replica service)

Reply:
Addressed in https://github.com/zalando/postgres-operator/pull/2161/commits/eea64b600cf1d2f09262772a69ce33ef4674119b